portal/config/config.py

```diff
@@ -103,6 +103,7 @@ class BaseConfig(object):
         ('assessment_cache_region', 60*60*2),
         ('reporting_cache_region', 60*60*12)]
     SEND_FILE_MAX_AGE_DEFAULT = 60 * 60  # 1 hour, in seconds
+    ENABLE_2FA = os.environ.get('ENABLE_2FA', None)
```

Comment:
@ivan-c is there a docker file in which we need to mention this var?  shouldn't be on anywhere but -test for now...

Reply:
We have a [`portal.env` "template"](https://github.com/uwcirg/truenth-portal/blob/develop/docker/portal.env.default) we could add it to, but I'm not entirely convinced we should add every value, effectively documenting them twice.

I might say this is important enough to do that for though.  :man_shrugging:

Reply:
agreed - no need for that level of detail.  thanks